org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/ZigBeeNetworkStateSerializerImpl.java

```diff
@@ -180,4 +180,16 @@ public void deserialize(final ZigBeeNetworkManager networkState) {
         logger.debug("Loading ZigBee network state: Done.");
     }
 
+    public synchronized void delete() {
+        final File file = new File(networkStateFilePath + "/" + networkStateFileName + networkId + ".xml");
+        boolean networkStateExists = file.exists();
+        if (networkStateExists) {
+            if (file.delete()) {
+                logger.debug("Deleting ZigBee network state: Done.");
```

Comment:
Suggestion: Align the debug messages for good and error case somewhat more; in particular, use the same name for the file in both (currently: 'ZigBee network state' vs. 'network state file').

Reply:
Will change the messages to be similar to those in the other two methods. In addition I will change `deserialize` to log the exception on error level instead of debug level.